common-js/src/resource-graph.ts

```diff
@@ -45,7 +45,7 @@ export const makeResourceGraphHelper = ({
 | where name startswith '${params.resourceNamePrefix}'
 | project name, type, location, subscriptionId, resourceGroup
 | join kind=leftouter (ResourceContainers | where type=='microsoft.resources/subscriptions' | project subscriptionName=name, subscriptionId) on subscriptionId
-| project name, location, resourceGroup, subscriptionId, subscriptionName, type`
+// | project name, location, resourceGroup, subscriptionId, subscriptionName, type`
```

Comment:
```suggestion
| project name, location, resourceGroup, subscriptionId, subscriptionName, type`
```

Reply:
Seems like this should be removed? Either the comment tag or the project line?

Reply:
Thanks. I did what you suggested.